build.gradle.kts

```diff
@@ -97,6 +97,13 @@ dependencies {
     testImplementation(imagej.core)
     testImplementation(imagej.ij)
     testImplementation(imgLib2.ij)
+
```

Comment:
Why do we need all of those libs? Aren't they at least partially transitive deps of your imgui port?